pbr/ensure_dependencies.py

```diff
@@ -9,27 +9,27 @@ def _install_pip():
     import requests
     import sys
 
+    install_file = os.path.join(os.path.dirname(__file__), "get_pip.py")
+
     # Get the get-pip.py install file
     print(
         "Pip is not installed, installing it now, restart the script after this is done"
     )
     try:
         r = requests.get("https://bootstrap.pypa.io/get-pip.py", stream=True)
         if r.status_code == 200:
-            with open("get_pip.py", "wb") as f:
+            with open(install_file, "wb") as f:
```

Comment:
This allows for running `ensure_dependencies.py` from the root.

Reply:
LGTM